Dockerfile

```diff
@@ -32,6 +32,9 @@ LABEL name="Red Hat Messagign QE - Java CLI Image" \
 
 USER root
 
+#Install fallocate util to create files of specific size
+RUN microdnf -y install \
+    util-linux
```

Comment:
I don't insist on `--setopt=install_weak_deps=0 --setopt=tsflags=nodocs`, but `&& microdnf clean all -y` probably should be there. Fedora dnf cache has like 80 MB

Reply:
I do, please follow the best practise from UBI, `microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y package && microdnf clean all `.